pkgs/applications/video/mpv/scripts/update-sponsorblock.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p git gnused jq nix-prefetch-git
+mpv_sponsorblock_git='/var/tmp/mpv_sponsorblock.git'
+
+set -euo pipefail
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+info() {
+    if [ -t 2 ]; then
+        set -- '\033[32m%s\033[39m\n' "$@"
+    else
+        set -- '%s\n' "$@"
+    fi
+    printf "$@" >&2
+}
+
+mpv_sponsorblock_old_rev=$(grep -Pom1 '(?<=\brev = ").*?(?=";)' sponsorblock.nix)
+mpv_sponsorblock_old_version=$(grep -Pom1 '(?<=\bversion = ").*?(?=";)' sponsorblock.nix)
```

Comment:
Can both be replaced by `nix-instantiate --strict --eval -A mpvScripts.sponsorblock.src.rev --json | jq -r` and mpvScripts.version respectively